ts-client/src/ValorClient.ts

```diff
@@ -6,9 +6,21 @@ import axios, { AxiosInstance } from 'axios';
  * @param value The value to type check.
  * @returns A boolean result.
  */
-function isGeoJSONObject(value: any): value is { type: string, coordinates: any } {
-  const geoJSONTypes: string[] = ["point", "linestring", "polygon", "multipoint", "multilinestring", "multipolygon"];
-  return typeof value === 'object' && value !== null && 'type' in value && geoJSONTypes.includes((value.type as string).toLowerCase());
+function isGeoJSONObject(value: any): value is { type: string; coordinates: any } {
```

Comment:
many of these changes were caused by running Prettier on this doc. I'll call-out actual functional changes via comments for convenience

Reply:
good with me!